### ISSUE TITLE
[bug report] kong startup Error

### ISSUE THREAD

**Issue by WALL-E** [Mashape/kong#1572]
### Summary

I find a bug, It could be caused by kong.conf, may also be caused by pl.config

### Steps To Reproduce

1. sudo yum install kong-0.9.0.*.noarch.rpm --nogpgcheck
2. cp /etc/kong/kong.conf.default kong.conf
3. Modify configuration file *kong.conf*

    ```
diff kong.conf.default kong.conf
103a104
> database = cassandra
159c160
< #cassandra_repl_factor = 1       # When migrating for the first time, Kong
---
> cassandra_repl_factor = 1       # When migrating for the first time, Kong
    ```
4. kong start

    ```
migrating core for keyspace kong
Error: Error during migration 2015-01-12-175310_skeleton: [Syntax error] line 2:82 no viable alternative at input '}' (...{'class': 'SimpleStrategy', 'replication_factor': [nil]}...)

  Run with --trace to see traceback
    ```

### Additional Details & Logs

- Kong version (0.9.0)
- CentOS Linux release 7.2.1511 (Core)

**Comment by WALL-E** [comment]
I did a little experiment to verify pl.config

## test code

* config.ini

```
# comment 1
name = zz # name, username

# comment 2
age = 18 # age

# comment 3
desc = this is a testa # desc
```

* t.lua

```
local config = require 'pl.config'
local pretty = require 'pl.pretty'

local t = config.read 'config.ini'

print(pretty.write(t))
```

## result

```
{
  name = {
    "zz # name",
    "username"
  },
  age = "18 # age",
  desc = "this is a testa # desc"
}
```

**Obviously, the result is not what we expect.**

**Comment by thibaultcha** [comment]
This should have been taken care of in #1569 if you want to give it a try.

**Comment by WALL-E** [comment]
fixed  #1569
